packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/useHandleUpload.ts

```diff
@@ -20,6 +30,11 @@ export interface CancelableTask extends Omit<Task, 'status'> {
   status: TaskStatus | 'CANCELED';
 }
 
+interface PendingTask {
+  task: CancelableTask;
+  result: Promise<any>;
```

Comment:
I think the `result` I'm using here is coming from `uploadData(input)`, so it's not the actual task or anything like that

Reply:
Why do we need it though? Interface changes need to well defined and explained

Reply:
oh gotcha, I think I added this because I was originally getting some typescript error for the return of my `processUpload` function and left it there. I just removed it and it looks like it's okay to remove